config/routes.rb

```diff
@@ -1,12 +1,16 @@
 Rails.application.routes.draw do
   root 'airlines#index'
 
-  resources :airlines, only:[:index, :new, :create, :show]
+  resources :airlines, only:[:index, :new, :create, :show, :post] do
+    resources :reviews
+  end
   devise_for :users
 
   namespace :api do
     namespace :v1 do
-      resources :airlines, only: [:show]
+      resources :airlines, only: [:show, :new, :create] do
+        resources :reviews
+      end
```

Comment:
You probably dont need create actions in your API controller and your regular controller for airlines
Additionally you should be using the `only` keyword with your resources :reviews

Reply:
you shouldnt need the `:new` of `r`esources :reviews, only: [:new, :create]` on line 12